core-services/release-controller/_releases/release-ocp-4.8.json

```diff
@@ -68,6 +68,10 @@
       "maxRetries": 3,
       "prowJob":{"name":"periodic-ci-openshift-release-master-nightly-4.8-e2e-metal-ipi"}
     },
+    "metal-ipi-serial-ipv4": {
```

Comment:
For consistency with the previous job I guess a better naming would be `metal-ipi-serial`